www/src/README.md

```diff
@@ -0,0 +1,12 @@
+
+## Running
+
+```bash
+roc run tutorial.roc -- input/ output/
+```
+
+## Serve files
+
+```bash
+python3 -m http.server --directory output/
+```
```

Comment:
Can you update this to use the rust `simple-http-server` like in the [recently updated README](https://github.com/roc-lang/roc/blob/871fa3bf3f2e2616d5f0d139a2688f3947c2d698/www/README.md)?

Reply:
Will do